source/_integrations/weatherflow.markdown

```diff
@@ -0,0 +1,65 @@
+---
+title: "WeatherFlow"
+description: "WeatherFlow weather sensor"
+ha_release: "0.38"
+ha_category: 
+  - Sensor
+ha_platforms:
+ - sensor
+ha_iot_class: "Local Polling"
+ha_quality_scale: silver
+ha_config_flow: true
+ha_codeowners:
+  - '@natekspencer'
+  - '@jeeftor'
+ha_domain: weatherflow
+---
+
+
+### WeatherFlow 
+
+The WeatherFlow integration uses the local UDP interface on a Weatherflow sensor, such as the Tempest, to discover and display weather data gathered vial the local network.
+
+### Installation Notes
+
+When you add this integration it may take a few minutes for all the entities to show up. The integration will listen for WeatherFlow UDP packets over the network and decode them as they arrive. As sensor-specific packets show up the sensor entity will be created.
+
+### Minor differences in calculated readings
+
+WeatherFlow sensors provide direct data for some sensors, where as other sensor values are the results of calculations. This integration utilizes [PsychroLib](https://github.com/psychrometrics/psychrolib) library to generate these calculations and as such there may be some drift from what is displayed in the Tempest Mobile App.
+
+
+# Sensors
+
+
+This integration will expose the following sensors:
+
+- Air Density
+- Dew Point
+- Feels Like
+- Humidity
+- Illuminance
+- Lightning Average Distance
+- Lightning Count
+- Precipitation Type
+- Rain Amount
+- Rain Rate
+- Solar Radiation
+- Station Pressure
+- Temperature
+- UV
+- Vapor Pressure
+- Wet Bulb Temperature
+- Wind Average
+- Wind Direction
+- Wind Direction Average
+- Wind Gust
+- Wind Lull
+- Wind Speed
+- Wind Speed Average
+
+Additionally it exposes the following Diagnostic Sensors:
```

Comment:
We do not need to explicitly make the difference between diagnostic and non-diagnostic sensors. That is just an UI trick and does not make them different otherwise. I suggest to merge the two lists.